### PR TITLE
Completed big o notation exercise

### PR DESCRIPTION
**Pull request by gregg-garceau**

### CODE REVIEW

File: big-o-notation/big-o.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(n)
+    i++                             // 2 * n = O(n)
+  ) {
+    const word = words[i];          // 3 * n = O(n)
+    let isUnique = true;            // 3 * n = O(n)
+    for (
+      let c = 0;                    // 1 * n^2 = O(n^2)
```
Comment (scbowler) [inline big-o-notation/big-o.js:30]: This line only runs once per the outer loop so it would be `O(n)`

File: big-o-notation/big-o.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      c < i;                        // 2 * n^2 = O(n^2)
+      c++                           // 2 * n^2 = O(n^2)
+    ) {
+      const comparing = words[c];   // 3 * n^2 = O(n^2)
```
Comment (scbowler) [inline big-o-notation/big-o.js:34]: This is 2 operations, access and assignment

File: big-o-notation/big-o.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      if (comparing === word) {     // 3 * n^2 = O(n^2)
+        isUnique = false;           // 3 * n^2 = O(n^2)
```
Comment (scbowler) [inline big-o-notation/big-o.js:36]: Assignment is 1 operation

File: big-o-notation/big-o.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    }
+    if (isUnique) {                 // 3 * n = O(n)
```
Comment (scbowler) [inline big-o-notation/big-o.js:39]: Truth check 1 operation

File: big-o-notation/big-o.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      unique[unique.length] = word; // 3 * n = O(n)
```
Comment (scbowler) [inline big-o-notation/big-o.js:40]: Length property access and assignment, so 2 operations here

**Review by scbowler — APPROVED** [review]
Looks good, a few mistakes on operations per line but overall looks like you got the main concepts.

File: big-o-notation/big-o.js (hunk 5 above)
Comment (gregg-garceau) [reply big-o-notation/big-o.js:40]: Sounds good! Ill fix these on my end

**Pull request merged**